### ISSUE TITLE
Fishing Boat Bonus- not working for anyone else.

### ISSUE THREAD

**Issue by VrogragFishslayer** [slowtorta/turtlewow-bug-tracker#5244]
The Fishing Boat summoned item is supposed to give +50 fishing to anyone in the boat. The +50 fishing bonus only applies to the person who put down the boat. 

1. Have an ally put down a boat
2. Step into boat
3. Note lack of fishing bonus

Item: https://database.turtle-wow.org/?item=51282
Spell: https://database.turtle-wow.org/?spell=46060

It should work as it has in the past, anyone in the boat should get the passive +50 fishing skill

Picture as reference

![boat2](https://github.com/slowtorta/turtlewow-bug-tracker/assets/139925005/c7d23ae5-cc5e-4165-b5ab-65f9e74eb8a6)

**Comment by twowTurtReynolds** [comment]
Is that boat right there, are you certain it was player summoned? Perhaps it is a scenery boat. I'll confirm on the PTR

**Comment by twowTurtReynolds** [comment]
Indeed was a player boat, the scenery boat I was thinking of was night elf themed and on the left.

Replicated the issue, and also, once the person who places it leaves the boat, they do not lose the skill. Even teleporting away cross continent keeps the buff going.

**Comment by VrogragFishslayer** [comment]
Yeah sorry I didn't get back to you on this. In the past leaving and re-entering the boat would allow you to keep the +50 AND other people could benefit as well. So I think something changed to the way the boat gives out the buff. And I knew that was a player boat because a player put one down for me (very thoughtful of them) but in the end it was useless to me. :-(

**Comment by roguestep16** [comment]
yeah and after 2 hours or so the buff is gone...
i thought thats working like it should^^ to be honest..
